### PR TITLE
Fix sriov functional tests for secure boot env

### PR DESCRIPTION
**Pull request by SchSeba**
This commit fix the functional tests when running on a secure boot environment

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: test/util/cluster/cluster.go [hunk 1]
```diff
@@ -113,8 +144,15 @@ func (n *EnabledNodes) FindOneMellanoxSriovDevice(node string) (*sriovv1.Interfa
...
+
+	// return error here as mlx interfaces are not supported when secure boot is enabled
+	// TODO: remove this when mlx support secure boot/lockdown mode
+	if n.IsSecureBootEnabled[node] {
+		return nil, fmt.Errorf("Secure boot is enabled on the node mellanox cards are not supported")
+	}
+
 	for _, itf := range s.Status.Interfaces {
-		if itf.Driver == "mlx5_core" {
+		if itf.Vendor == mlxVendorID {
```
[the remark this answers is not in the document]
Comment (fedepaol) [reply test/util/cluster/cluster.go:156]: Thanks!

**Comment by SchSeba** [comment]
@mmirecki please take a look

**Pull request merged**